src/components/SelectionPills/SelectionPill/SelectionPill.d.ts

```diff
@@ -22,16 +22,16 @@ export interface SelectionPillProps extends WithThemeInjectedProps {
   isDisabled?: boolean
 
   /** Callback function called after pill click */
-  onClick(): void
+  onClick?(e: React.MouseEvent<HTMLElement>, props: SelectionPillProps): void
 
   /** Callback function called after pill gained focus */
-  onFocus(): void
+  onFocus?(e: React.MouseEvent<HTMLElement>, props: SelectionPillProps, isFocused: boolean): void
 
   /** Callback function called after pill has lost focus */
-  onBlur(): void
+  onBlur?(e: React.MouseEvent<HTMLElement>, props: SelectionPillProps, isFocused: boolean): void
```

Comment:
For these, I think props and focus are both properties in an object that is the second parameter.